rst_files/kalman.rst

```diff
@@ -666,29 +619,22 @@ Exercise 1
 
 .. code-block:: julia
 
-    using Distributions
-
     # parameters
-
     θ = 10
     A, G, Q, R = 1.0, 1.0, 0.0, 1.0
     x_hat_0, Σ_0 = 8.0, 1.0
 
     # initialize Kalman filter
-
     kalman = Kalman(A, G, Q, R)
     set_state!(kalman, x_hat_0, Σ_0)
-    # == Run == #
 
-    N = 5
     xgrid = range(θ - 5, θ + 2, length = 200)
-    densities = []
-    labels = []
-    for i ∈ 1:N
+    densities = Array{Array{Float64, 1}, 1}(undef, 5) # we'll do 5 rounds of updating
```

Comment:
Style no concrete.

Reply:
This one's a bit tough because it's a container of arrays, so `zeros` doesn't apply. 

I changed it to:

```
temp = [ [1.0], [2.0], [3.0], [4.0], [5.0] ]
densities = similar(temp); # we'll do 5 rounds of updating
```